spec/services/random_phrase_spec.rb

```diff
@@ -21,5 +21,13 @@
 
       expect(phrase.to_s.length).to eq 24 # 20 chars + 4 spaces
     end
+
+    it 'does not contain the letters I L O U' do
+      100.times do
```

Comment:
on my machine the entire test file runs under a second, so that seems fast enough to me.